doc/api/deprecations.md

```diff
@@ -0,0 +1,574 @@
+   memory.
+* [`Buffer.from(array)`][] - Create a `Buffer` with a copy of `array`
+* [`Buffer.from(arrayBuffer[, byteOffset[, length]])`][from_arraybuffer] - Create a `Buffer`
+  that wraps the given `arrayBuffer`.
+* [`Buffer.from(buffer)`][] - Create a `Buffer` that copies `buffer`.
+* [`Buffer.from(string[, encoding])`][from_string_encoding] - Create a `Buffer` that copies
+  `string`.
+
+<a id="DEP0006"></a>
+### DEP0006: child\_process options.customFds
+
+Type: Runtime
+
+Within the [`child_process`][] module's `spawn()`, `fork()`, and `exec()`
+methods, the `options.customFds` option is deprecated. The `options.stdio`
+option should be used instead.
+
+<a id="DEP0007"></a>
+### DEP0007: cluster worker.suicide
+
+Type: Runtime
+
+Within the `cluster` module, the [`worker.suicide`][] property has been
+deprecated. Please use [`worker.exitedAfterDisconnect`][] instead.
+
+<a id="DEP0008"></a>
+### DEP0008: require('constants')
+
+Type: Documentation-only
+
+The `constants` module has been deprecated. When requiring access to constants
+relevant to specific Node.js builtin modules, developers should instead refer
+to the `constants` property exposed by the relevant module. For instance,
+`require('fs').constants` and `require('os').constants`.
+
+<a id="DEP0009"></a>
+### DEP0009: crypto.pbkdf2 without digest
+
+Type: Runtime
+
+Use of the [`crypto.pbkdf2()`][] API without specifying a digest is deprecated.
+Please specify a digest.
+
+<a id="DEP0010"></a>
+### DEP0010: crypto.createCredentials
+
+Type: Runtime
+
+The [`crypto.createCredentials()`][] API is deprecated. Please use
+[`tls.createSecureContext()`][] instead.
+
+<a id="DEP0011"></a>
+### DEP0011: crypto.Credentials
+
+Type: Runtime
+
+The `crypto.Credentials` class is deprecated. Please use [`tls.SecureContext`][]
+instead.
+
+<a id="DEP0012"></a>
+### DEP0012: Domain.dispose
+
+Type: Runtime
+
+[`Domain.dispose()`][] is deprecated. Recover from failed I/O actions
+explicitly via error event handlers set on the domain instead.
+
+<a id="DEP0013"></a>
+### DEP0013: fs async function without callback
+
+Type: Runtime
+
+Calling an asynchronous function without callback is deprecated.
+
+<a id="DEP0014"></a>
+### DEP0014: fs.read legacy String interface
+
+Type: Runtime
+
+The [`fs.read()`][] legacy String interface is deprecated. Use the Buffer API as
+mentioned in the documentation instead.
+
+<a id="DEP0015"></a>
+### DEP0015: fs.readSync legacy String interface
+
+Type: Runtime
+
+The [`fs.readSync()`][] legacy String interface is deprecated. Use the Buffer
+API as mentioned in the documentation instead.
+
+<a id="DEP0016"></a>
+### DEP0016: GLOBAL/root
+
+Type: Runtime
+
+The `GLOBAL` and `root` aliases for the `global` property have been deprecated
+and should no longer be used.
+
+<a id="DEP0017"></a>
+### DEP0017: Intl.v8BreakIterator
+
+Type: Runtime
+
+The `Intl.v8BreakIterator` is deprecated and will be removed or replaced soon.
```

Comment:
The replacement is currently a [stage 2 proposal](https://github.com/tc39/proposal-intl-segmenter)